src/validators/typed_dict.rs

```diff
@@ -184,6 +185,9 @@ impl BuildValidator for TypedDictValidator {
                 default_factory,
                 on_error,
                 frozen: field_info.get_as::<bool>(intern!(py, "frozen"))?.unwrap_or(false),
+                validate_always: field_info
+                    .get_as::<bool>(intern!(py, "validate_always"))?
```

Comment:
as with `frozen`, this should be configurable on the schema - as we do with `Settings`.

Reply:
you mean add `typed_dict_frozen` and `typed_dict_validate_always`?

Reply:
My mistake, wrong PR, i meant like https://github.com/pydantic/pydantic-core/pull/228#discussion_r944560977 we should use `schema_or_config_same(schema, config, "validate_always")` here (here we use [`schema_or_config_same`](https://github.com/pydantic/pydantic-core/blob/6d431dc5f78b567f8e7c8975291edc08a7ce2baa/src/build_tools.rs#L84) since I think the names should be the same).

(But yes, I guess we should add a config setting for `frozen` too, but that can be a separate issue & PR)

Reply:
#231